pkg/clusteragent/externalmetrics/autoscaler_watcher_test.go

```diff
@@ -54,7 +54,7 @@ func newAutoscalerFixture(t *testing.T) *autoscalerFixture {
 	}
 }
 
-func (f *autoscalerFixture) newAutoscalerWatcher() (*AutoscalerWatcher, kube_informer.SharedInformerFactory, dynamic_informer.DynamicSharedInformerFactory) {
+func (f *autoscalerFixture) newAutoscalerWatcher(onlyUseDatadogMetrics bool) (*AutoscalerWatcher, kube_informer.SharedInformerFactory, dynamic_informer.DynamicSharedInformerFactory) {
```

Comment:
I don't think the parameter is necessary. The test code can update the `AutoscalerWatcher` object field directly when necessary (only one test)

Reply:
Will adjust